lib/src/RadioButton/RadioGroup/RadioGroup.tsx

```diff
@@ -1,9 +1,9 @@
 import React from "react";
```

Comment:
Can you please move this component one level back? we don't need to have another folder called RadioGroup to host it. Just drop it at RadioButton. We still haven't decided how to organize helper components but as of now, this will only create confusion when the users want to import the component directly and not from the root